build-tools/packages/build-cli/src/commands/report/codeCoverage.ts

```diff
@@ -158,8 +175,18 @@ export default class ReportCodeCoverageCommand extends BaseCommand<
 		);
 
 		// Fail the build if the code coverage analysis shows that a regression has been found.
-		if (!success) {
+		if (!(success || skipBuildFailureOnRegression)) {
 			this.error("Code coverage failed", { exit: 255 });
 		}
 	}
 }
+
+const summaryFooterOnFailure =
+	"### What to do if the code coverage check fails:\n" +
+	"- In the code coverage summary, a checkbox will appear which will not allow the code coverage check to fail the build. Now, if the regression is due to removal of tests, removal of code with lots of tests or any other valid reason, please tick the checkbox and trigger the build again. This till it will still do the comparison, but will not fail the build due to regression.\n" +
+	"- You can again uncheck the checkbox, to fail the build on regression before triggering the build.\n" +
+	"- You can also add more tets to increase the code coverage for the package which showed the regression in the summary.\n" +
+	"- You may sometimes want to check which lines are covered or not covered by your tests, you can follow these steps:\n" +
+	"  - Go to the PR ADO build.\n" +
+	"  - Then, go to the published artifacts. You will see a `codeCoverageAnalysis` named artifact, which you can expand to reach to a particular file's coverage html which will show which lines are covered/not covered by your tests.\n" +
+	"  - You can also run different kind of tests locally with `:coverage` tests commands to find out the coverage.\n";
```

Comment:
Thoughts? Is it still clear? If we go with flipping `skip<>` to `fail<>` we'll want to update here a bit.

Besides this, I think we can make things a bit simpler if in lines 162-168 we always include this text (not just on failure) and have the checkbox always read "[ ] Skip build failure in case of test-coverage regression. See below for details on what this does.", and only use `success` to set the checked/unchecked status of the box.

```suggestion
	"### What to do if the code coverage check fails:\n" +
	"Ideally, add more tests to increase the code coverage for the package(s) whose code-coverage regressed.\n\n" +
	"If a regression is causing the build to fail and is due to removal of tests, removal of code with lots of tests or any other valid reason, there is a checkbox further up in this comment that determines if the code coverage check should fail the build or not. You can check the box by editing this comment and trigger the build again. The test coverage analysis will still be done, but it will not fail the build if a regression is detected.\n" +
	"Unchecking the checkbox (again by editing the comment) and triggering another build should go back to failing the build if a test-coverage regression is detected.\n\n" +
	"You can check which lines are covered or not covered by your tests with these steps:\n" +
	"- Go to the PR ADO build.\n" +
	"- Click on the link to see its published artifacts. You will see an artifact named `codeCoverageAnalysis`, which you can expand to reach to a particular source file's coverage html which will show which lines are covered/not covered by your tests.\n" +
	"- You can also run different kind of tests locally with `:coverage` tests commands to find out the coverage.\n";
```

Reply:
I have most of it except for "editing the comment" part. To check/uncheck the box, you don't need to edit comment. 
Also improved the structure but the kept the words.